src/controllers/pallets/PalletsNominationPoolsController.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { PalletsNominationPoolService } from '../../services';
+import AbstractController from '../AbstractController';
+
```

Comment:
Hey, great observation @Imod7. So the comments on the controllers were legacy docs before we have the OpenAPI specs a long time ago. So there is no need anymore for the docs in the actual controller itself. 

So that being said, I would also be totally ok with removing all the comments out of the controllers in general and leave the source of truth to the OpenAPI specs. If that's something you'd like to tackle let me know. For know I will make an issue about it.